models/classes/Platform/Repository/Lti1p3RegistrationRepository.php

```diff
@@ -133,12 +115,38 @@ private function getDefaultPlatform(): Platform
             self::PLATFORM_ID,
             self::PLATFORM_ID,
             rtrim(ROOT_URL, '/'),
-            self::OIDC_URL
+            self::OIDC_URL,
+            self::OAUTH_URL
         );
     }
 
     private function getLtiProviderService(): LtiProviderService
     {
         return $this->getServiceLocator()->get(LtiProviderService::SERVICE_ID);
     }
+
+    private function createRegistrationByProvider(LtiProvider $ltiProvider): ?Registration
+    {
+        $toolKeyChain = $this->getToolKeyChainRepository()
+                ->findAll(new KeyChainQuery($ltiProvider->getId()))
+                ->getKeyChains()[0] ?? null;
+
+        $platformKeyChain = $this->getPlatformKeyChainRepository()
+                ->findAll(new KeyChainQuery())
+                ->getKeyChains()[0] ?? null;
+
+        if ($toolKeyChain === null || $platformKeyChain === null) {
+            return null;
```

Comment:
i think `return` will return null by default so maybe you don't need the `null` specified

Reply:
It needs to be null, cause of return type" `?Registration`.

Reply:
```
return; ===  return null;
```
it still return null in both cases :)

Reply:
AS we discussed, in this case is necessary. We cannot use just `return` or it will break. The `return null` is mandatory :)